app/assets/javascripts/local_time.js.coffee

```diff
@@ -200,7 +200,8 @@ update = (callback) ->
   document.addEventListener "time:elapse", callback
 
   if Turbolinks?.supported
-    document.addEventListener "page:update", callback
+    for event in ["page:update", "turbolinks:load"]
+      document.addEventListener event, callback
```

Comment:
IMO we should remove the Turbolinks hooks entirely and provide a MutationObserver that automatically processes `<time>` elements.